src/vma/main.cpp

```diff
@@ -109,6 +109,8 @@ const char *vma_version_str = "VMA_VERSION: " PACKAGE_VERSION "-" STR(VMA_LIBRAR
 #endif
 			      ;	// End of vma_version_str - used in "$ strings libvma.so | grep VMA_VERSION"
 
+#define VMA_GIT_TITLE		"Git: "
+const char *vma_git_version_str = VMA_GIT_TITLE  VMA_GIT_VERSION;
```

Comment:
Probably there is no need in using this global variable for single print. Please consider move one to `print_vma_global_setting()` or use direct `VMA_GIT_VERSION` inside printf call.